service/notifier.go

```diff
@@ -250,3 +253,28 @@ func (p *notifier) decodeSearchAttrBinary(bytes []byte, key string) interface{}
 	}
 	return val
 }
+
+func (p *notifier) sendMessageToTestServer(notification *Notification) error {
```

Comment:
The issue is for sending to actual subscribers? https://github.com/cadence-oss/cadence-notification/blob/bf9062942ec12764c118fc3fc21bb4debf3b9583/common/config/Config.go#L82

I am fine if you want to have a separate PR for thst

Reply:
let me update it using Webhook